AppBuilder/platform/ABObjectApi.js

```diff
@@ -1,10 +1,70 @@
+const atomicjs = require("atomicjs");
```

Comment:
I don't sure .

Do we need to use this `atomicjs` library ?

Does it possible if we use `node's fetch` function instead ?
https://nodejs.org/dist/latest-v18.x/docs/api/globals.html#fetch